tests/org.codechecker.eclipse.rcp.it.tests/build.properties

```diff
@@ -2,4 +2,5 @@ source.. = src/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
-               fragment.xml
+               fragment.xml,\
+               resources/
```

Comment:
Resources now will be included in the integration test bundle/jar. A test project already added in this pr.